src/App.tsx

```diff
@@ -3,9 +3,77 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { getMovie } from './api';
+import { MovieData } from './types/MovieData';
+import { ResponseError } from './types/ReponseError';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState<string>('');
+  const [isError, setIsError] = useState<boolean>(false);
```

Comment:
you should not specify simple types
```suggestion
  const [query, setQuery] = useState<>('');
  const [isError, setIsError] = useState<>(false);
```